backend/data_import/pipeline/examples/article_annotation/example.json

```diff
@@ -0,0 +1,74 @@
+[
+    {
+        "text": "the text to be annotated",
+        "cats": "general category/categories for the given text",
+        "entities": "specific label/labels for a specific word or a span of words",
+        "relations": "relation labels between entities",
+        "meta": {
+            "article_title": "title of article",
+            "publish_datetime": "publication datetime of article",
+            "crawled_datetime": "datetime when our program collected the article",
+            "type": "title/lead/paragraph/subheader/comment"
+        },
+        "article_id": "8eabas8s5pveszg917vm1357",
+        "order": 1
+    },
+    {
+        "text": "Stół z powyłamywanymi nogami",
+        "cats": ["sadness", "negative"],
+        "entities": [[0, 3, "rzeczownik"], [7, 27, "narzędnik"]],
+        "relations": [],
+        "meta": {
+            "article_title": "10 Craziest Polish Tongue Twisters",
+            "publish_datetime": "2022-08-01 13:59:59",
+            "crawled_datetime": "2022-09-05 08:12:45",
+            "type": "subheader"
+        },
+        "article_id": "8eabas8s5pveszg917vm1357",
+        "order": 5
+    },
+    {
+        "text": "W Szczebrzeszynie chrząszcz brzmi w trzcinie.",
+        "cats": ["joy", "positive"],
+        "entities": [
+            {
+                "id": 0,
+                "start_offset": 2,
+                "end_offset": 16,
+                "label": "miejscownik"
+            },
+            {
+                "id": 1,
+                "start_offset": 18,
+                "end_offset": 26,
+                "label": "rzeczownik"
+            },
+            {
+                "id": 2,
+                "start_offset": 28,
+                "end_offset": 32,
+                "label": "czasownik"
+            }
+        ],
+        "relations": [
+            {
+                "from_id": 1,
+                "to_id": 2,
+                "type": "isDoing"
+            },
+            {
+                "from_id": 0,
+                "to_id": 1,
+                "type": "explainsWhere"
+            }
+        ],
+        "meta": {
+            "article_title": "10 Craziest Polish Tongue Twisters",
+            "publish_datetime": "2022-08-01 13:59:59",
+            "crawled_datetime": "2022-09-05 08:12:45",
+            "type": "subheader"
+        },
+        "article_id": "8eabas8s5pveszg917vm1357",
+        "order": 10
+    }
```

Comment:
I think we can remove this last array element from this file, and put it in the backend/data_import/pipeline/examples/custom_relation_extraction/example.json (or jsonl) file. Because it is to be an example of data with relation XD